lib/services/rally/rally_hierarchical_requirement_resource.rb

```diff
@@ -41,9 +41,19 @@ def create_path(element_name)
     end
   end
 
+  # Ensure that we pass in the workspace parameter on create, so that
+  # we are routed to the correct API endpoint
+  def add_workspace_param_to_url url
```

Comment:
It might be worth wrapping the argument in parentheses here to match the rest of the file.